### DISCUSSION TITLE
Discussions continuing?

### DISCUSSION THREAD

**Discussion by MikeMcC399** [corona-warn-app/cwa-website#3405] [Development] [answered]
Hopefully discussions will stay enabled in this repository. It would be helpful to have some up-to-date information from the Open Source Team about this.

At one point it was said they would be shut down, however they are still enabled and there was even a new addition https://github.com/corona-warn-app/cwa-website/discussions/3401 recently made by converting an issue into a discussion.

If they are going to be disabled, then I would avoid creating any new discussion, so I would like to know what the plans are.

**Comment by larswmh** [comment, marked as answer]
I unfortunately cannot tell for certain whether Discussions will stay enabled or not, but if I had to guess it will probably remain enabled for the remaining timespan. From my perception it has proven itself as quite useful for both maintainers and contributors.

Either way, please don't feel discouraged to create any new discussion if you would like to. Even if Discussions were to be disabled, your effort would not be lost as it would most likely just be converted to an issue, like it was before we had Discussions.

I have converted the issue you mentioned to a discussion in the announcement category in an effort to reduce our amount of open issues, particularly because this specific one is not really an issue and rather an information.
